c++/include/orc/Reader.hh

```diff
@@ -39,6 +39,29 @@ namespace orc {
   struct ReaderOptionsPrivate;
   struct RowReaderOptionsPrivate;
 
+  struct ReaderMetrics {
+    volatile uint64_t ReaderCount;
+    volatile uint64_t ReaderInclusiveLatencyUs;
+    volatile uint64_t DecompCount;
+    volatile uint64_t DecompLatencyUs;
+    volatile uint64_t DecodingCount;
+    volatile uint64_t DecodingLatencyUs;
+    volatile uint64_t ByteDecodingCount;
+    volatile uint64_t ByteDecodingLatencyUs;
+
+    ReaderMetrics() {
+      ReaderCount = 0;
```

Comment:
nit: provide the default value in the declaration so we can avoid explicit constructor here.

Reply:
has removed the explicit constructor.